examples/evaluation/lora-evaluation.ipynb

```diff
@@ -0,0 +1,4369 @@
+      "vocab.txt: 100%|█████████████████████████████| 213k/213k [00:00<00:00, 3.89MB/s]\n",
+      "tokenizer.json: 100%|████████████████████████| 436k/436k [00:00<00:00, 19.8MB/s]\n",
+      "/opt/conda/lib/python3.10/site-packages/transformers/tokenization_utils_base.py:1617: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be deprecated in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n",
+      "  warnings.warn(\n",
+      "model.safetensors: 100%|██████████████████████| 436M/436M [00:02<00:00, 163MB/s]\n",
+      "If you want to use `BertLMHeadModel` as a standalone, add `is_decoder=True.`\n",
+      "hellaswag.py: 100%|████████████████████████| 4.36k/4.36k [00:00<00:00, 29.3MB/s]\n",
+      "dataset_infos.json: 100%|██████████████████| 2.53k/2.53k [00:00<00:00, 22.2MB/s]\n",
+      "README.md: 100%|███████████████████████████| 6.84k/6.84k [00:00<00:00, 22.6MB/s]\n",
+      "Downloading data: 100%|█████████████████████| 47.5M/47.5M [00:00<00:00, 143MB/s]\n",
+      "Downloading data: 100%|█████████████████████| 11.8M/11.8M [00:00<00:00, 214MB/s]\n",
+      "Downloading data: 100%|█████████████████████| 12.2M/12.2M [00:00<00:00, 201MB/s]\n",
+      "Generating train split: 100%|████| 39905/39905 [00:06<00:00, 6412.01 examples/s]\n",
+      "Generating test split: 100%|█████| 10003/10003 [00:01<00:00, 6441.44 examples/s]\n",
+      "Generating validation split: 100%|█| 10042/10042 [00:01<00:00, 6313.93 examples/\n",
+      "Map: 100%|███████████████████████| 39905/39905 [00:08<00:00, 4696.95 examples/s]\n",
+      "Map: 100%|███████████████████████| 10042/10042 [00:02<00:00, 4204.44 examples/s]\n",
+      "100%|███████████████████████████████████| 10042/10042 [00:04<00:00, 2159.16it/s]\n",
+      "Running loglikelihood requests:   0%|                 | 0/40168 [00:00<?, ?it/s]We strongly recommend passing in an `attention_mask` since your input_ids may be padded. See https://huggingface.co/docs/transformers/troubleshooting#incorrect-output-when-padding-tokens-arent-masked.\n",
+      "Running loglikelihood requests: 100%|████| 40168/40168 [04:56<00:00, 135.59it/s]\n",
+      "fatal: not a git repository (or any parent up to mount point /kaggle)\n",
+      "Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n",
+      "hf (pretrained=bert-base-cased,dtype=bfloat16), gen_kwargs: (None), limit: None, num_fewshot: None, batch_size: 128\n",
+      "|  Tasks  |Version|Filter|n-shot| Metric |   |Value |   |Stderr|\n",
+      "|---------|------:|------|-----:|--------|---|-----:|---|-----:|\n",
+      "|hellaswag|      1|none  |     0|acc     |↑  |0.2490|±  |0.0043|\n",
+      "|         |       |none  |     0|acc_norm|↑  |0.2458|±  |0.0043|\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!lm_eval \\\n",
```

Comment:
I'm wondering: Since we're in an interactive Python session, can we not use `lm_eval` as Python package instead of shelling out? I'm not familiar with the package but it looks like calling `lm_eval.evaluator.simple_evaluate` should have the same effect.